simphony/io/tests/test_h5_lattice.py

```diff
@@ -73,9 +108,10 @@ def cleanup(self):
         shutil.rmtree(self.temp_dir)
 
     def container_factory(self, name, type_, base_vect, size, origin):
-        return H5Lattice.create_new(
-            self.handle.root, 'foo', type_, base_vect,
-            size, origin, record=CustomRecord)
+        self.group = self.handle.create_group(
```

Comment:
Yes, it does not provide anything additional for the tests since the `handle.root` is already a group. On the contrary, using `handle.root`, makes sure that the `H5Lattice` proxy class does not depended on specific structures outside of the provided group node since any attempt to access a node before the `handle.root` with fail.